### PR TITLE
DB-5633: fix timestamp column for system tables

### PR DESCRIPTION
**Pull request by jyuanca**
Timestamp column for system tables is encoded by V1 serializer, but is decoded by v2 serializer. In general, you get incorrect values for a timestamp column if you do a select on the column. Fix this problem by using a V1 serializer to decode system tables.

### CODE REVIEW

File: db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/BaseDataDictionary.java [hunk 1]
```diff
@@ -52,6 +52,9 @@
 	protected static final int		SYSCOLUMNS_CORE_NUM = 2;
 	protected	static final int		SYSSCHEMAS_CORE_NUM = 3;
 	protected static final int        NUM_CORE = 4;
+	// NOTE: JC - When adding a new sys table, this
+	// number will need to be increased in BOTH places.
+	public static final long FIRST_USER_TABLE_NUMBER = 1458;
```
Comment (dgomezferro) [inline db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/BaseDataDictionary.java]: This constant is also defined in DataDictionary, I think we should use just that one and remove this.

File: db-shared/src/main/java/com/splicemachine/db/shared/common/reference/SQLState.java [hunk 2]
```diff
@@ -1942,5 +1942,6 @@ The message identifier (ie. the value that matches a field in this class)
 	String BACKUP_CANCELED                                         = "BR011";
 	String BACKUP_DOESNOT_EXIST                                    = "BR012";
 	String PARENT_BACKUP_MISSING                                   = "BR013";
+	String EXISTS_CONCURRENT_BACKUP                                = "BR014";
```
Comment (dgomezferro) [inline db-shared/src/main/java/com/splicemachine/db/shared/common/reference/SQLState.java]: Isn't this change unrelated?

File: db-tools-i18n/src/main/resources/com/splicemachine/db/loc/messages.xml [hunk 3]
```diff
@@ -8841,6 +8841,11 @@ Shutting down instance {0} on database directory {1} with class loader {2} </tex
                <arg>backupId</arg>
                <arg>parentBackupId</arg>
            </msg>
+           <msg>
+               <name>BR014</name>
+               <text>A concurrent {0} backup is running.</text>
+               <arg>backupType</arg>
+           </msg>
```
Comment (dgomezferro) [inline db-tools-i18n/src/main/resources/com/splicemachine/db/loc/messages.xml]: Unrelated.

File: db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/BaseDataDictionary.java (hunk 1 above)
Comment (jyuanca) [inline db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/BaseDataDictionary.java]: Yes, I did not find that. Thanks!

File: db-shared/src/main/java/com/splicemachine/db/shared/common/reference/SQLState.java (hunk 2 above)
Comment (jyuanca) [inline db-shared/src/main/java/com/splicemachine/db/shared/common/reference/SQLState.java]: A code clean up.
